tests/suites/test_suite_psa_its.function

```diff
@@ -1,6 +1,8 @@
 /* BEGIN_HEADER */
 #include "../library/psa_crypto_its.h"
 
+#define PSA_ASSERT( expr ) TEST_EQUAL( ( expr ), PSA_SUCCESS )
```

Comment:
I agree that the way I did it is not ideal, but I don't like your proposal better than what I did. I find it less clear because it works by general case and exception and it has negations of negations.

I think `psa_helpers` should be broken up into `psa_helpers` with only `#include "psa/error.h" and a definition of `PSA_ASSERT`, and `psa_crypto_helpers` with `#include "psa/crypto.h"` and all the rest. And perhaps even a `psa_crypto_se_driver_helpers` for secure element test drivers and so on. But this feels out of proportion with what we're doing now, and we don't even have `psa/error.h` yet.